changelog

```diff
@@ -33,6 +33,8 @@ pandoc (2.7)
     Note that you can use `--pdf-engine-opt=-outdir=bar` to specify
     a persistent temp directory.
 
+  * Add `tectonic` as an option for `--pdf-engine` (#5345).
```

Comment:
Better not to change the changelog in the PR, since you don't know whether it will be included in this release.